spec/models/minion_spec.rb

```diff
@@ -4,29 +4,61 @@
 describe Minion do
   it { is_expected.to validate_uniqueness_of(:hostname) }
 
+  # rubocop:disable RSpec/ExampleLength
```

Comment:
The fact that you have to disable this is already telling you that this method is too complex 😉